pkgs/development/cuda-modules/overrides/default.nix

```diff
@@ -0,0 +1,9 @@
+{ lib }:
+lib.mapAttrs'
+  # Strip the file extension from the name if present, and import the file/directory.
+  (name: value: {
+    name = lib.removeSuffix ".nix" name;
+    value = import (./. + "/${name}");
+  })
+  # Make sure to remove this file (default.nix) from the list of files/directories.
+  (lib.removeAttrs (builtins.readDir ./.) [ "default.nix" ])
```

Comment:
Now we're pulling nits, but I suspect there's a complexity difference between this and `concatMap` xDDD